### PR TITLE
Add Discord profile view button on user details page

### PR DESCRIPTION
**Pull request by sahsisunny**
### Closes : #381 

1. Show full name of the user if present 
2. Add button to see discord profile of the user 

**Before**
image

**After**
image

### CODE REVIEW

File: users/details/script.js [hunk 1]
```diff
@@ -72,7 +72,7 @@ function generateUserImage(alt) {
 }
 
 function createSocialMediaAnchorNode({ href, id, alt, src }) {
-  const a = createElement({ type: 'a', classList: ['social'] });
+  const a = createElement({ type: 'a', classList: ['social', alt] });
   a.setAttribute('target', '_blank');
   a.setAttribute('rel', 'noopener noreferrer');
   a.setAttribute('href', `${href}/${userData[id] ? userData[id] : ''}`);
```
Comment (Pratiyushkumar) [inline users/details/script.js:78]: use proper variable names please

File: users/details/style.css [hunk 2]
```diff
@@ -20,6 +20,7 @@ html {
   --blue-hover-color: #0d0a4d;
   --razzmatazz-color: #e30464;
   --razzmatazz-hover-color: #a1034a;
+  --gray-color: gray;
 }
```
Comment (Pratiyushkumar) [inline users/details/style.css:24]: color hex code please, please follow the pattern of the code base

File: users/details/style.css [hunk 3]
```diff
@@ -82,10 +83,18 @@ li {
   width: 100px;
 }
```
Comment (Pratiyushkumar) [inline users/details/style.css:84]: please use rem instead of px

**Review by Pratiyushkumar — CHANGES_REQUESTED** [review]

**Review by shubhamsigdar1 — COMMENTED** [review]
great work @sahsisunny...I written one comment please reply

**Comment by shubhamsigdar1** [comment]
Nit: Can You make a video...in which you are clicking discord link and what we see...just want to see :)

File: users/details/style.css (hunk 2 above)
Comment (sahsisunny) [reply users/details/style.css:24]: It's not necessary. That's why we are using variables

File: users/details/style.css (hunk 3 above)
Comment (sahsisunny) [reply users/details/style.css:84]: These changes are not mine.

File: users/details/script.js (hunk 1 above)
Comment (sahsisunny) [reply users/details/script.js:78]: https://github.com/Real-Dev-Squad/website-dashboard/pull/380#discussion_r1233337244

File: users/details/script.js [hunk 4]
```diff
@@ -72,7 +72,7 @@ function generateUserImage(alt) {
 }
 
 function createSocialMediaAnchorNode({ href, id, alt, src }) {
-  const a = createElement({ type: 'a', classList: ['social'] });
+  const a = createElement({ type: 'a', classList: ['social', alt] });
```
[the remark this answers is not in the document]
Comment (sahsisunny) [reply users/details/script.js:75]: To identify the anchor tag

**Comment by sahsisunny** [comment]
> Nit: Can You make a video...in which you are clicking discord link..just want to see :)

Not needed because we added the short description.

**Review by Pratiyushkumar — APPROVED** [review]
LGTM

File: users/details/script.js [hunk 5]
```diff
@@ -112,15 +112,35 @@ function generateUserData(userData) {
...
     classList: ['user-details-wrap'],
   });
-  const username = createElement({
+  const fullName = createElement({
     type: 'h2',
+    classList: ['user-details-fullname'],
+  });
+  fullName.appendChild(
+    createTextNode(
+      userData?.first_name + ' ' + userData?.last_name ?? userData.username,
```
Comment (khushi818) [inline users/details/script.js:121]: I think it will replace lastname with username if not found. check it once

**Review by khushi818 — CHANGES_REQUESTED** [review]

File: users/details/script.js (hunk 5 above)
Comment (sahsisunny) [reply users/details/script.js:121]: Not needed, it's create confusion

Comment (khushi818) [reply users/details/script.js]: Thanks for the changes

**Review by khushi818 — APPROVED** [review]

**Review by shubhamsigdar1 — APPROVED** [review]
LGTM

**Review by j24m — APPROVED** [review]

**Review by Pratiyushkumar — APPROVED** [review]
LGTM

**Pull request merged**